engine/execution/computation/computer/customblocks_test.go

```diff
@@ -105,7 +122,7 @@ func (vmt vmTest) run(
 		ledger.On("Prove", mock.Anything).
 			Return(expectedProof, nil).
 			Times(numTxPerCol*numCol + 1)
-		committer := committer.NewLedgerViewCommitter(ledger, trace.NewNoopTracer())
+		committer := committer.NewLedgerViewCommitter(ledger, tracer)
```

Comment:
added the unmocked tracer here, not totally sure if it needs its own tracer or not